added hello world in Urdu language, account # 255f824a504702ed4c11a71a5efabee085754940b3eb42e3677a5813ad29585f

saifi369 opened the pull request:
account # 255f824a504702ed4c11a71a5efabee085754940b3eb42e3677a5813ad29585f

Hristijan95 commented:
@saifi369 , there was already a "Hello World" in Urdu language, can you please explain why is it wrong and what's the difference?

saifi369 commented:
First of all, sorry I couldn't reply on time.
I added the "Hello World" again because the first one was not the TRANSLATION of "Hello World" in my native language, it was how we pronounce the "Hello World". I added the translation of "Hello World" in Urdu language.
